lib/everest.cpp

```diff
@@ -486,6 +486,13 @@ void Everest::subscribe_var(const Requirement& req, const std::string& var_name,
         EVLOG_verbose << fmt::format(
             "Incoming {}->{}", this->config.printable_identifier(requirement_module_id, requirement_impl_id), var_name);
 
+        // A race condition where some other module might received its on-ready
+        // and this module not (and calling us).
+        while (!ready_received) { // In C++20 we might mark it as [[unlikely]]
+            EVLOG_warning << "Not yet ready";
+            std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
maybe we can reduce this sleep time here to something in the hundreds-of-ms range and at least store it in a constant that can be reused for the other subscriptions. making it configurable (and maybe considering a timeout) would be perfect